packages/media/src/types/resource.ts

```diff
@@ -77,20 +77,13 @@ export interface Resource {
   /** Whether the resource externally hosted. */
   isExternal: boolean;
   /** Whether the resource is a placeholder. */
-  isPlaceholder?: boolean;
+  isPlaceholder: boolean;
   /** Whether the resource needs a CORS proxy. */
-  needsProxy?: boolean;
+  needsProxy: boolean;
   /** Resource creation date. */
   readonly creationDate?: string;
   /** Resource sizes */
-  sizes: { [key: string]: ResourceSize } | never[];
+  sizes: { [key: string]: ResourceSize };
   /** Resource author attribution */
   attribution?: Attribution;
-
-  // TODO: This property should probably not end up in a story. Only relevant for the media library.
-  provider?: 'local' | 'unsplash' | 'coverr' | 'tenor' | 'tenor_stickers';
-
-  // TODO: Figure out why sometimes _images_ end up having these properties.
-  posterId?: ResourceId;
-  isOptimized?: boolean;
```

Comment:
Also added because of the templates IIRC.

If we remove this here, we should also remove these properties from the affected images in the templates.